src/utils.ts

```diff
@@ -0,0 +1,68 @@
+import { ContributionPartyEvent } from "types";
+import { ethersProvider } from "./ethereum";
+import {
+  haveSetNewPartyWithContribution,
+  haveSetPartyHalfway,
+  setNewPartyWithContribution,
+  setPartyHalfway,
+} from "./party_events";
+
+export const bestUserName = async (address: string): Promise<string> => {
+  const ens = await ethersProvider.lookupAddress(address);
+  if (ens) {
+    return ens;
+  } else {
+    return address;
+  }
+};
+
+export const getIsNewPartyWithContribution = async (
```

Comment:
nit but to me it'd be more natural to name this someething like`shouldAlertAboutNewParty`

Reply:
i typically prepend `get` to methods that return booleans because `isFuncName` or `shouldFuncName` names could be confused as variables. what do you think?

Reply:
i like the `should` prefix but its not a huge deal